tests/README.md

```diff
@@ -1,5 +1,7 @@
 # Testing istio.io Content
 
+**NOTE: We are now migrating to a new test framework. The following content is updated for the new framework.**
+
```

Comment:
We are going to do a one time switch, no migrating, so you don't need to say anything about it in here.
```suggestion
```